_data/meltano/utilities/superset/apache.yml

```diff
@@ -0,0 +1,32 @@
+name: superset
```

Comment:
We shouldn't merge this until we have a have first-user instructions around creating an admin user, overriding `secret_key` etc.